react/DiscountBadge.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react'
+import { FormattedNumber } from 'react-intl'
+import PropTypes from 'prop-types'
+
+/**
+* The discount badge component. It receives the product's list and selling prices
+* and calculates the discount percent to show it in the product's sumary.
+*/
+class DiscountBadge extends Component {
+  /**
+  * This method calculates the discount tax by the selling and list prices.
+  */
+  calculateDiscountTax() {
+    const listPrice = this.props.listPrice
```

Comment:
Aqui você pode descontruir os `props`, acho que fica mais legal.
`const {listPrice, sellingPrice} = this.props` 
http://es6-features.org/#ObjectMatchingShorthandNotation

Reply:
Adicionei isso, realmente fica mais bacana.